modelbaker/iliwrapper/ili2dbconfig.py

```diff
@@ -297,6 +299,12 @@ def to_ili2db_args(self, extra_args=[], with_action=True):
         elif self.db_ili_version is None or self.db_ili_version > 3:
             self.append_args(args, ["--strokeArcs=False"])
 
+        if self.tool & DbIliMode.gpkg:
+            if self.create_gpkg_multigeom:
+                self.append_args(args, ["--gpkgMultiGeomPerTable"], True)
+            elif self.db_ili_version is None or self.db_ili_version > 3:
+                self.append_args(args, ["--gpkgMultiGeomPerTable=False"])
```

Comment:
More a question than a suggestion here:

Why do we pass this `"--gpkgMultiGeomPerTable=False"` ?
Why not just omitting it? Is it for the sake of making things more explicit?

Reply:
It's about the metaconfiguration we can pass to ili2db with parameters (maybe you remember this part in the UsabILIty Hub onboarding). In case there is smart1inheritance, it's concerned but if we add additionally to the ili2db command smart2inhertiance, the "manual" parameter overrides the one from the metaconfig. Now if a parameter is here (true) or not here (false) and if it's contained in the metaconfig, we might want to unset it. Instead of modify the metaconfiguration file we just set it false. And this is what it's done here. When it's false by the setting, then it should be set to false (and not just be omitted).

Reply:
Thank you!